openhands/runtime/action_execution_server.py

```diff
@@ -241,7 +247,7 @@ def _resolve_path(self, path: str, working_dir: str) -> str:
     async def read(self, action: FileReadAction) -> Observation:
         # NOTE: the client code is running inside the sandbox,
         # so there's no need to check permission
-        working_dir = self.bash_session.workdir
+        working_dir = self.bash_session.pwd
```

Comment:
`pwd` technically stands for `print working directory`, so this might not be the best name. `cwd` is more accurate (`current working directory`)

Reply:
fixed in most recent commit!